### ISSUE TITLE
Question : accumulate_grad_batches seems not working

### ISSUE THREAD

**Issue by amejri** [NVIDIA/NeMo#3767]
Hi guys,

I'm trying to train conformer rnnt model with accumulating several batches for gradient. For that, I specified it in the trainer as follow : 
```
    trainer = pl.Trainer(gpus=gpus, max_epochs=nb_epochs, distributed_backend="ddp", precision=16, accumulate_grad_batches=32)
```
but it seems like nothing changes. and the training evolution is the same as without adding accumulate_grad_batches parameter.
Do you have any idea how to solve it ?

With kind regards,

**Comment by VahidooX** [comment]
What sort of changes do you expect when you use accumulate_grad_batches?
It is handled mostly by PTL and less unlikely to have a bug. Have you tried other models like CTC models?
